lib/Table/TableColumn.ts

```diff
@@ -252,6 +252,30 @@ export default class TableColumn {
       }
     };
 
+    let yyyyQQ: StringToDateFunction = value => {
+      // Is it quarterly data in the format yyyy-Qx ? (Ignoring null values, and failing on any purely numeric values)
+      if (value.indexOf("-Q") === 4) {
+        var year = value.slice(0, 4);
+        var quarter = value.slice(6);
+        var monthString;
+        if (quarter === "1") {
+          monthString = "01/01";
+        } else if (quarter === "2") {
+          monthString = "04/01";
+        } else if (quarter === "3") {
+          monthString = "07/01";
+        } else if (quarter === "4") {
+          monthString = "10/01";
+        } else {
+          parsingFailed = true;
+          return null;
+        }
+        return new Date(year + "/" + monthString);
```

Comment:
Maybe we should also handle the situation where `year` is invalid and the whole date object becomes invalid.

Reply:
done!